You can pass an `Error` object to `captureException()` to get it captured as event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stack trace.

```javascript{tabTitle: Angular 12+}
import * as Sentry from "@sentry/angular-ivy";

try {
  aFunctionThatMightFail();
} catch (err) {
  Sentry.captureException(err);
}
```

```javascript{tabTitle: Angular 10/11}
import * as Sentry from "@sentry/angular";

try {
  aFunctionThatMightFail();
} catch (err) {
  Sentry.captureException(err);
}
```
